packages/miniflare/src/plugins/assets/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const ASSETS_PLUGIN_NAME = "assets";
+export const ASSETS_SERVICE_NAME = `${ASSETS_PLUGIN_NAME}:assets-service`;
+export const ROUTER_SERVICE_NAME = `${ASSETS_PLUGIN_NAME}:router`;
+export const FAKE_KV_SERVICE_NAME = `${ASSETS_PLUGIN_NAME}:kv`;
```

Comment:
personally I'd rather stick with the existing naming convention that we have in Miniflare, which doesn't seem to use prefixing such as `FAKE_`. I don't see any added value to that prefix. I think `KV_SERVICE_NAME` or `ASSETS_KV_SERVICE_NAME` should do the trick

Reply:
ASSETS_KV_SERVICE_NAME sounds good to me :)

Reply:
much awesomeness!